DPGAnalysis/HcalNanoAOD/python/customiseHcalCalib_cff.py

```diff
@@ -0,0 +1,59 @@
+import FWCore.ParameterSet.Config as cms
+
+# Customization for running on testEnablesEcalHcal
+#   - Call from cmsDriver.py with: `--customise DPGAnalysis/HcalNanoAOD/customiseHcalCalib_cff.customiseHcalCalib`
+def customiseHcalCalib(process):
+    # Add uMNio digi (special digi identifies calib event type)
+    process.load("DPGAnalysis.HcalNanoAOD.hcalUMNioTable_cff")
+    process.hcalNanoTask.add(process.uMNioTable)
+    process.hcalNanoDigiTask.add(process.uMNioTable)
+
+    process.load("DPGAnalysis.HcalNanoAOD.hcalUHTRTable_cff")
+    process.hcalNanoTask.add(process.uHTRTable)
+    process.hcalNanoDigiTask.add(process.uHTRTable)
+
+    # Raw data has a different name, hltHcalCalibrationRaw instead of rawDataCollector
+    process.hcalDigis.InputLabel = cms.InputTag('hltHcalCalibrationRaw')
+    process.uHTRTable.InputLabel = process.hcalDigis.InputLabel
+    
+    # Create EDFilter for HLT_HcalCalibration
+    # (HCAL raw data is not present in ECAL-triggered events, annoyingly. The filter stops downstream modules from throwing ProductNotFound.)
+    process.hcalCalibHLTFilter = cms.EDFilter("TriggerResultsFilter",
+        triggerConditions = cms.vstring(
+          'HLT_HcalCalibration_v6 / 1'),
```

Comment:
Do you really need to specific version of trigger here? This will make the code more complicated as version of trigger can change in future.

Reply:
Can it just be 'HLT_HcalCalibration_v*'?

Reply:
Thanks, this does work. I made the change. (Somehow I thought it didn't accept wildcards.)